src/storybook/stand-alone-documentaion/accessibility/accessibility.stories.mdx

```diff
@@ -1,13 +1,188 @@
-import { Meta } from "@storybook/addon-docs";
-import { ComponentName } from "../../components";
-import { ComingSoon } from "../../components/coming-soon/coming-soon";
+import {Meta} from "@storybook/addon-docs";
+import {ComponentName, Frame, UnstyledList, UnstyledListItem} from "../../components";
+import {PrinciplesAccesability} from "./principles/principlesAccesability";
+import {
+    accessibilityCheckList,
+    colorContrastImage,
+    colorContrastText,
+    hierarchyOrder,
+    imageryDecorative,
+    imageryDescriptive,
+    imageryAlt,
+    navigationFocus,
+    navigationTabs
+} from "./assets";
 
 <Meta
-  title="Foundations/Accessibility"
+    title="Foundations/Accessibility"
 />
 
 <ComponentName withFoundationBackground>
-  Accessibility
+    Accessibility
 </ComponentName>
 
-<ComingSoon />
\ No newline at end of file
+- [Intro](#intro)
+- [Accessibility principles](#accessibility-principles)
+- [Color contrast](color-contrast)
+- [Imagery](#Imagery)
+- [Hierarchy and focus](#hierarchy-and-focus)
+- [Navigation](#navigation)
+- [Assistive Technology](#assistive-technology)
+- [Feedback](#feedback)
+
+## Intro
+Web Accessibility is focused on ensuring people with disabilities can equally perceive, understand, navigate, and interact with digital content. It also means that they can contribute equally like anyone else.
+Web Accessibility is based on guidelines published by The World Wide Web Consortium (W3C) for accessible content called Web Content Accessibility Guidelines or WCAG.
+
+[WCAG Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize#principle1)
+
+<Tip title="Get our Accessibility checklist"><a href={accessibilityCheckList} target="_blank">Download</a> our accessibility checklist file.</Tip>
+
+## Accessibility principles
+<PrinciplesAccesability/>
+
+## Color contrast
+__Text Color Contrast__<br/>
+Applies to text in the page, including placeholder text, hover text and text shown on keyboard focus.
+The visual presentation of text and images should have a contrast ratio of at least 4.5:1 with the background
+
+<Frame noBorder noGutter>
+    <img src={colorContrastText} alt="example for text color contrast"/>
```

Comment:
If the user cannot see the picture I think the alt text will not be useful for him, no?
Shouldn't it be hidden maybe from non-sight users?

Reply:
if the user cannot see the picture , he can still understand there is a context. this is why we use alt.
but as demonstrated below, we can also use role="presentation" but i do not think this is the case.

what do you think?

Reply:
ok let's keep it this way :)